metrics/network/network-metrics-iperf3.sh

```diff
@@ -75,10 +94,29 @@ function iperf3_bidirectional_bandwidth_client_server {
 	local client_command="mount -t ramfs -o size=20M ramfs /tmp && iperf3 -c ${server_address} -d -t ${time}"
 	start_client "$extra_args" "$client_name" "$image" "$client_command" > "$result"
 
-	local total_bidirectional_client_bandwidth=$(cat $result | tail -n 3 | head -1 | awk '{print $(NF-2), $(NF-1)}')
-	local total_bidirectional_server_bandwidth=$(cat $result | tail -n 4 | head -1 | awk '{print $(NF-3), $(NF-2)}')
-	echo "Network bidirectional bandwidth (client to server) is : $total_bidirectional_client_bandwidth"
-	echo "Network bidirectional bandwidth (server to client) is : $total_bidirectional_server_bandwidth"
+	local client_result=$(grep -m1 -E '\breceiver\b' $result)
+	local server_result=$(grep -m1 -E '\bsender\b'   $result)
+	local -a client_results
+	read -a client_results <<< ${client_result}
+	read -a server_results <<< ${server_result}
+	local total_bidirectional_client_bandwidth=${client_results[6]}
+	local total_bidirectional_client_bandwidth_units=${client_results[7]}
+	local total_bidirectional_server_bandwidth=${server_results[6]}
+	local total_bidirectional_server_bandwidth_units=${server_results[7]}
+	echo "Network bidirectional bandwidth (client to server) is :" \
+		"$total_bidirectional_client_bandwidth" \
+		"$total_bidirectional_client_bandwidth_units"
+	echo "Network bidirectional bandwidth (server to client) is :" \
+		"$total_bidirectional_server_bandwidth" \
+		"$total_bidirectional_server_bandwidth_units"
+
+	save_results "${test_name}" "network bidir bw client to server" \
```

Comment:
I've mulled over this a bit. I'm going to make the call to keep the data together for the minute in a single file. Now that you extract more of the results from some of the tests, I think if we put those into separate files then there is some risk in the future that we'd end up with a set of files and no proof that the data came from the same test run (well, OK, maybe timestamps and commit IDs could give a heavy clue... still...). So, let's leave as is, and then we will fix up `checkmetrics` and related other scripts to pull out the data they are looking for.